pysmt/solvers/cvc.py

```diff
@@ -0,0 +1,534 @@
+#
```

Comment:
The filename is inconsistent, we are calling the solver `cvc5` but the new files do not have the number.
I wonder if just calling the solver CVC would be better at this point.